### PR TITLE
Navigator and Search: add .stream files by guessing

### PR DESCRIPTION
**Pull request by nsacerdote**
Proof of concept, simple approach to add stream files in navigator tree and search (by guessing their path)

Related with #23

### CODE REVIEW

File: src/main/java/com/shade/decima/ui/dialogs/FindFileDialog.java [hunk 1]
```diff
@@ -199,6 +196,16 @@ private List<FileInfo> buildFileInfoIndex(@NotNull Project project) throws IOExc
         return info;
     }
 
+    private Map<Long, List<Packfile>> buildFileHashToPackfilesMap(PackfileManager manager) {
```
Comment (ShadelessFox) [inline src/main/java/com/shade/decima/ui/dialogs/FindFileDialog.java:199]:
```suggestion
    @NotNull
    private Map<Long, List<Packfile>> buildFileHashToPackfilesMap(@NotNull PackfileManager manager) {
```

File: src/main/java/com/shade/decima/ui/dialogs/FindFileDialog.java [hunk 2]
```diff
@@ -164,28 +164,25 @@ private List<FileInfo> buildFileInfoIndex(@NotNull Project project) throws IOExc
...
             final String path = PackfileBase.getNormalizedPath(file.get("Path"));
             final long hash = PackfileBase.getPathHash(path);
-
             for (Packfile packfile : packfiles.getOrDefault(hash, Collections.emptyList())) {
                 info.add(new FileInfo(packfile, path, 0));
+                containing.add(hash);
             }
 
-            containing.add(hash);
+            final String streamPath = PackfileBase.getNormalizedPath(file.get("Path")) + ".stream";
```
Comment (ShadelessFox) [inline src/main/java/com/shade/decima/ui/dialogs/FindFileDialog.java:180]:
```suggestion
            final String streamPath = path + ".stream";
```

**Review by ShadelessFox — COMMENTED** [review]
Are there cases when the `xxx.core` has its stream counterpart called `xxx.core.stream`? In most cases, their names vary, don't they?

**Comment by nsacerdote** [comment]
> Are there cases when the `xxx.core` has its stream counterpart called `xxx.core.stream`? In most cases, their names vary, don't they?

At least in HzD using this approach I'm able to guess 33266 .stream files

**Comment by ShadelessFox** [comment]
I see, cool!

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**